enot_config.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "mongodb",
-  "app_vsn": "3.1.1",
+  "app_vsn": "3.1.2",
```

Comment:
don't know why they are different, but 3.3.2 is correct. Can you please change?

Reply:
good catch. honestly I don't know what `enot_config.json` is even for but I figured the versions should match.

correct to `3.3.2` in 436e9c0

Reply:
enot is another build system, rebar alternative.